extensions/odoo_theme/static/js/menu.js

```diff
@@ -57,4 +57,18 @@
      * Sidebar receives the `o-mobile-show` class.
      */
 
+     /**
+     * Scroll listener: used to hide searchbar + will be useful for "Go to top" button.
+     **/
+
+     document.addEventListener('scroll', () => {
+        let scroll = this.scrollY;
+        const header = document.querySelector('.o_main_header');
+        if (scroll >= 1) {
+            header.classList.add('o_header_scroll');
+        } else {
+            header.classList.remove('o_header_scroll');
+        }
+     });
```

Comment:
```suggestion
     document.addEventListener('scroll', () => {
         // Allow to hide the searchbar when the page is scrolled in mobile.
         _flagHeaderWithScrollPosition();
     });
     
    /**
     * Add/Remove the class `o_header_scrolled` on the header according to the scroll position.
     */
     const _flagHeaderWithScrollPosition = () => {
         const header = document.querySelector('.o_main_header');
         if (this.scrollY > 0) {
             header.classList.add('o_header_scrolled');
         } else {
             header.classList.remove('o_header_scrolled');
         }
     });
```
- Extract this behavior in a separate function so that it's easier to add other scroll handlers.
- Format the docstring according to the guidelines.
- Use `const` when the variable is never assigned another value. If the variable is only used once, it doesn't need to be assigned at all (except for clarity reasons).
- `> 0` is more obvious than `>= 1` IMO.
- Use 4 spaces tabulations.
- Suggestion: replace `o_header_scroll` with `o_header_scrolled` to denotated a current state.